shell/platform/android/io/flutter/embedding/engine/loader/FlutterLoader.java

```diff
@@ -0,0 +1,342 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.embedding.engine.loader;
+
+import android.content.Context;
+import android.content.pm.ApplicationInfo;
+import android.content.pm.PackageManager;
+import android.content.res.AssetManager;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Looper;
+import android.os.SystemClock;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.util.Log;
+import android.view.WindowManager;
+
+import io.flutter.BuildConfig;
+import io.flutter.embedding.engine.FlutterJNI;
+import io.flutter.util.PathUtils;
+import io.flutter.view.VsyncWaiter;
+
+import java.io.File;
+import java.util.*;
+
+/**
+ * This class finds Flutter resources in an application APK and also loads Flutter's native library.
```

Comment:
nit: Should be a fragment.

```suggestion
 * Finds Flutter resources in an application APK and also loads Flutter's native library.
```

https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment

Reply:
Done.